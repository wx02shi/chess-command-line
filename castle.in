game human human
setup
+ K e1
+ R a1
= black
+ K e8
+ R h8
done
move e1 c1
move e8 g8
resign
