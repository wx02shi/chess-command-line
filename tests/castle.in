game human human
setup
+ R a8
+ K e8
done
move e8 a8
move a8 e8