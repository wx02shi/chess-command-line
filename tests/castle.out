8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh
8  _ _k_ r
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh
Setup completed!
8  _ _k_ r
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ KR_ _ 

  abcdefgh
8  _ _ rk_
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ KR_ _ 

  abcdefgh
Final Score:
White: 0
Black: 1
